src/explore-education-statistics-frontend/src/modules/table-tool/components/TableToolFinalStep.tsx

```diff
@@ -216,75 +153,32 @@ const TableToolFinalStep = ({
-                <li>
-                  Publication:{' '}
-                  <Link
-                    to="/find-statistics/[publication]"
-                    as={`/find-statistics/${selectedPublication.slug}`}
-                  >
-                    {selectedPublication.title},{' '}
-                    {selectedPublication.selectedRelease.title}
-                  </Link>
-                </li>
-              ) : (
-                <li>
-                  Publication:{' '}
-                  <Link
-                    to="/find-statistics/[publication]/[releaseSlug]"
-                    as={`/find-statistics/${selectedPublication.slug}/${selectedPublication.selectedRelease.slug}`}
-                  >
-                    {selectedPublication.title},{' '}
-                    {selectedPublication.selectedRelease.title}
-                  </Link>
-                </li>
-              )}
-              {publication?.methodology?.slug && (
-                <li>
-                  Methodology:{' '}
-                  <Link
-                    to="/methodology/[methodology]"
-                    as={`/methodology/${publication.methodology.slug}`}
-                  >
-                    {publication.methodology.title}
-                  </Link>
-                </li>
-              )}
-              {publication?.externalMethodology?.url && (
-                <li>
-                  Methodology:{' '}
-                  <a href={publication.externalMethodology.url}>
-                    {publication.externalMethodology.title}
-                  </a>
-                </li>
-              )}
-            </ul>
-          </AccordionSection>
-        )}
-        {publication?.contact && (
-          <AccordionSection heading="Contact us">
-            <p>
-              If you have a question about the data or methods used to create
-              this table contact the named statistician:
-            </p>
-            <h4 className="govuk-heading-s govuk-!-font-weight-bold">
-              {publication?.contact.teamName}
-            </h4>
-            <p>Named statistician: {publication?.contact.contactName}</p>
-            <p>
-              Email:{' '}
-              <a href={`mailto:${publication?.contact.teamEmail}`}>
-                {publication?.contact.teamEmail}
-              </a>
-            </p>
-            <p>Telephone: {publication?.contact.contactTelNo}</p>
-          </AccordionSection>
-        )}
-      </Accordion>
+      <TableToolInfo
+        table={table}
+        contactDetails={publication?.contact}
+        methodologyLink={getMethodologyLink()}
+        releaseLink={
+          <>
+            {selectedPublication.selectedRelease.latestData ? (
+              <Link
+                to="/find-statistics/[publication]"
+                as={`/find-statistics/${selectedPublication.slug}`}
+              >
+                {selectedPublication.title},{' '}
+                {selectedPublication.selectedRelease.title}
```

Comment:
Would suggest avoid splitting strings into separate nodes as it makes it harder for Robot tests to select these elements based on their text i.e. do this instead:

```tsx
{`${selectedPublication.title}, ${selectedPublication.selectedRelease.title}`}
```